lib/rust/ensogl/component/text/src/component/area.rs

```diff
@@ -571,15 +587,19 @@ impl AreaModel {
         let scene = &app.display.default_scene;
         let logger = Logger::new("text_area");
         let selection_map = default();
-        let fonts = scene.extension::<typeface::font::Registry>();
-        let font = fonts.load("DejaVuSansMono");
-        let glyph_system = typeface::glyph::System::new(&scene, font);
+        #[cfg(target_arch = "wasm32")]
+        let glyph_system = {
+            let fonts = scene.extension::<typeface::font::Registry>();
+            let font = fonts.load("DejaVuSansMono");
+            typeface::glyph::System::new(&scene, font)
+        };
```

Comment:
if you mock `fonts.load` instead, then this file will be way nicer, am I right?

Reply:
You can't instantiate `glyph::System` anyway, so it does not make sense to instantiate (mocked) `Font` as well. Mocking `Font` is totally possible, but will only lead to more changes in this PR without any functional difference.